spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/repository/query/SpannerQueryLookupStrategyTests.java

```diff
@@ -0,0 +1,109 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.data.spanner.repository.query;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import org.springframework.cloud.gcp.data.spanner.core.SpannerOperations;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.SpannerMappingContext;
+import org.springframework.data.repository.core.NamedQueries;
+import org.springframework.data.repository.query.QueryMethod;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.same;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Chengyuan Zhao
+ */
+public class SpannerQueryLookupStrategyTests {
+
+	private SpannerOperations spannerOperations;
+
+	private SpannerMappingContext spannerMappingContext;
+
+	@Before
+	public void initMocks() {
+		this.spannerOperations = mock(SpannerOperations.class);
+		this.spannerMappingContext = new SpannerMappingContext();
+	}
+
+	@Test
+	public void resolveSqlQueryTest() {
```

Comment:
nit: can you please extract the mock QueryMethod creation to a separate method and use that in both tests?

Reply:
done!!